tools/fuchsia/build_fuchsia_artifacts.py

```diff
@@ -112,6 +112,12 @@ def CopyGenSnapshotIfExists(source, destination):
                     destination_base, 'kernel_compiler.snapshot')
 
 
+def CopyFlutterTesterBinIfExists(source, destination):
+  source_root = os.path.join(_out_dir, source)
+  destination_base = os.path.join(destination, 'flutter_binaries')
+  FindFileAndCopyTo('flutter_tester', source_root, destination_base)
```

Comment:
Can the path to the flutter_tester be explicitly specified? AFAIK, there are multiple Flutter tester binaries in the out out directory (a stripped and unstripped one).

Reply:
Good call, we currently do this for `gen_snapshot` and related binaries too. I am reworking packaging to account for debug symbols and multiple target hosts. Will fix it as a part of a follow up PR.

Currently, this should still get us to a place where we have all the binaries needed by Fuchsia so can remove the flutter/engine dep from topaz.